Makefile

```diff
@@ -79,3 +79,5 @@ gometalinter: ## Run gometalinter on all go files
 help:  ## Show help messages for make targets
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}'
 
+verify: depend vet gofmt
```

Comment:
Do we want to just rename `check` job to `verify` and verify boilerplate there? Is there anything against using lint in basic verify checks or that should be another job?

Reply:
Sounds great!